src/bfx_tools/topiary.ml

```diff
@@ -0,0 +1,198 @@
+open Biokepi_run_environment
+open Common
+
+type output_type = [
+  | `HTML of string
+  | `CSV of string
+]
+
+type predictor_type = [
+  | `NetMHC
```

Comment:
Also, it's worth pointing out that these options come from mhctools. Should that be wrapped also?